packages/adblocker/test/parsing.test.ts

```diff
@@ -876,6 +883,30 @@ describe('Network filters', () => {
       });
     });
 
+    describe('removeparam', () => {
+      it('parses ~removeparam', () => {
+        // ~removeparam is not a valid option
+        network('||foo.com$~removeparam=utm', null);
+        network('||fo.com$~removeparam', null);
+      });
+
+      it('parses removeparam', () => {
+        network('||foo.com$removeparam=utm', {
+          removeparam: 'utm',
+          isRemoveParam: true,
+          isRedirectable: true,
+        });
+      });
+
+      it('parses removeparam without a value', () => {
```

Comment:
Does this mean to remove all query parameters?

Reply:
Exactly